dist/components/progress-steps/_progress-steps.dust

```diff
@@ -0,0 +1,19 @@
+{!}
+
+    Progress Component Template
+    ===========================
+
+    Progress steps are intended to give visibility as to where one is in a multi step process.
+
+{!}
+
+{#progressStep}
```

Comment:
I don’t think we want this wrapping key: it should be the responsibility of the developer to make sure the correct keys are available in the template without setting a context here, I think.